packages/app/src/cli/models/app/app.ts

```diff
@@ -17,6 +17,7 @@ export enum WebType {
 
 export const WebConfigurationSchema = schema.define.object({
   type: schema.define.enum([WebType.Frontend, WebType.Backend]),
+  authCallbackPath: schema.define.string().default(''),
```

Comment:
What about making it optional `.optional()` instead of defaulting to an empty string. I think it's safer from the consumer side because doesn't have to remember about the potential emptiness of the string. If it's `undefined`.